### ISSUE TITLE
how to use multi-gpus to finetune large datasets like ReCoRD?

### ISSUE THREAD

**Issue by runzeer** [nyu-mll/jiant#1338]
**Describe the bug**

**To Reproduce**
1. Tell use which version of `jiant` you're using
2. Describe the environment where you're using `jiant`, e.g, "2 P40 GPUs"
3. Provide the experiment config artifact (e.g., `defaults.conf`)

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by zphang** [comment]
The main runscript should automatically detect the number of GPUs available and run with DataParallel. Are you encountering any issues doing so?

**Comment by runzeer** [comment]
when using the DataParallel, the memory cost in each GPU is not equal. So if convenient, could you change the mode to DDP?

**Comment by zphang** [comment]
Yep, we are aware that alternative multi-GPU methods like DDP have some advantages over DataParallel, but switching to DDP will introduce non-trivial complexity, and we do not currently have plans to incorporate it.
